protocols/raft/src/main/java/io/atomix/protocols/raft/impl/DefaultRaftMetadataClient.java

```diff
@@ -62,7 +62,7 @@ public MemberId getLeader() {
 
   @Override
   public Collection<MemberId> getServers() {
-    return selectorManager.servers();
+    return selectorManager.members();
```

Comment:
@kuujo was this intentional or should the getter be renamed too?

Reply:
hmm yeah missed a rename there